src/sitemap/toc.rs

```diff
@@ -95,6 +97,7 @@ impl TocItemCompat {
         readers: Vec<String>,
         writers: Vec<String>,
         icon: Option<String>,
+        bury: Option<bool>,
```

Comment:
Change the type to bool. `bury: true` or `bury: false` is not specified, assume `bury: false` is there.

Reply:
Please review https://github.com/ftd-lang/fpm/pull/622 for the same.